app/views/pages/home.html.erb

```diff
@@ -53,6 +53,28 @@ technical talent shortage. -->
   </div>
   <br><br>
 
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12" role="main">
+        <p class="lead">
+          In the United States over 200,000 computing jobs open annually and 30,000 are filled by Computer Science graduates; 
+          however, that number is expected to rise to 1.2 million by 2020. Meanwhile, we have 250,000 U.S. military personnel that exit the service annually, including some because of sequestration.  Currently, there are almost <a href="http://www.bls.gov/news.release/pdf/vet.pdf">500,000 unemployed veterans</a> with over 70,000 of us between the ages of 25 and 34, which is 
+          the typical age of separation, as of 2015 according to the Bureau of Labor Statistics. 
+          </br>
+          When you join our ranks you'll be joining our growing military veterans coding community. In <a href="http://operation-code.slack.com/">Slack</a>,
+          you'll network, share resources, find job opportunities, get help with your code, and become eligible for conference scholarships and coding 
+          scholarships. Together we are Operation Code.
+        </p>
+      </div>
+    </div>
+  </div>
+  <div class='col m12 center'>
+   <!-- <div class="row">
+      <!-- Location for future sign-up form -->
```

Comment:
Sorry to be a pain but I think we can just straight up remove this. The less code the better!